themes/default/data/registry/packages/talos.yaml

```
@@ -0,0 +1,14 @@
+category: Infrastructure
+component: false
+description: A Pulumi package for creating and managing Talos Linux machines and clusters.
+featured: false
+logo_url: https://www.talos.dev/images/Sidero_stacked_darkbkgd_RGB.png
+name: talos
+native: false
+package_status: public_preview
+publisher: Pulumiverse
+repo_url: https://github.com/pulumiverse/pulumi-talos
+schema_file_path: provider/cmd/pulumi-resource-talos/schema.json
+title: talos
+updated_on: 1698780654
+version: v0.1.7
```

If we want to bump to 0.1.8, this should do it:

```suggestion
updated_on: 1699915346
version: v0.1.8
```

---

@cnunciato preferably not do this. I made improvements to the docs also. The scheduled check will copy these over too.